src/renderers/shaders/ShaderLib/depth_vert.glsl

```diff
@@ -11,4 +13,6 @@ void main() {
 	#include <logdepthbuf_vertex>
 	#include <clipping_planes_vertex>
 
+	viewZDepth = - mvPosition.z;
```

Comment:
Minor issue...

Usually we use the following pattern and the `v` nomenclature for varyings:

    vViewPosition = - mvPosition.xyz;

So, for consistency, you could name it `vViewDistance` or `vViewDepth`.